src/components/Charts/ModelChart.js

```diff
@@ -26,116 +30,140 @@ const ModelChart = ({
   interventions,
   currentIntervention,
 }) => {
-  const { isEmbed } = useEmbed();
-  const interventionToModel = {
-    [INTERVENTIONS.LIMITED_ACTION]: interventions.baseline,
-    [INTERVENTIONS.SOCIAL_DISTANCING]:
-      interventions.distancingPoorEnforcement.now,
-    [INTERVENTIONS.SHELTER_IN_PLACE]: interventions.distancing.now,
-  };
-
-  let model = interventionToModel[currentIntervention];
+  let interventionsModelIsValid = true;
+  let model;
 
-  if (currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE) {
-    model = interventionToModel[INTERVENTIONS.SOCIAL_DISTANCING];
+  // In future we can do further model shape/schema checking
+  try {
+    let interventionToModel = interventionToModelMap(interventions);
+    let worstCaseIntervention = getWorstCaseIntervention(currentIntervention);
+    model = interventionToModel[worstCaseIntervention];
```

Comment:
this is pulling the logic for "worst case" social distancing:
```
  if (currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE) {
    model = interventionToModel[INTERVENTIONS.SOCIAL_DISTANCING];
  }
```
 that's used in like 3 places in the app into the one helper function to keep things from getting out of sync